helm/dagster/charts/dagster-user-deployments/values.yaml

```diff
@@ -9,6 +9,11 @@ dagsterHome: "/opt/dagster/dagster_home"
 postgresqlSecretName: "dagster-postgresql-secret"
 celeryConfigSecretName: "dagster-celery-config-secret"
 
+# Mount the Dagster instance configuration from the main chart's configmap.
+# Set to true when user deployments are in the same namespace as the main Dagster chart.
+# Set to false when user deployments are in a different namespace to avoid mount issues.
+mountInstanceConfig: false
```

Comment:
naming nit - I think 'mount' is how its implemented but i think the user here more just cares about whether its included. Maybe includeInstance?

It might also be good to explain why somebody would do this (as i understand it, to use features that require access to the instance at import/startup time)